job_templates/sklearn_svm/config_fed_server.conf

```diff
@@ -0,0 +1,54 @@
+{
+  format_version = 2
+  server.task_request_interval = 0.05
```

Comment:
we don't normally see this, might be good to add comments to explain what this is for?